.buildkite/full_pipeline_trigger.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env sh
+
+if [[ "$BUILDKITE_MESSAGE" == *"[QUICK CI]"* ]]; then
```

Comment:
Do we want this to be possible? It would allow developers to skip the full checks on next/master.

Reply:
Possibly not, just added it as a utility.

Reply:
Hmm, slightly in two minds here as Github status checks should cover us.  I'm not sure how much we'd actually want to use it and if there is a compelling case for it.

Reply:
Just thought we have cases where a changelog gets changed or typo gets fixed before a release and we often have to wait for a full CI run for that minor change.  Would it be acceptable to use quick-ci in these cases?

Reply:
With a new commit the Github status checks will be reset, so we'd need another full run _at some point_ before being able to merge - but perhaps I can see the use...  If there are a few review comments to action (including code changes) and the author wants to address them in separate commits with fast-feedback, it could be useful to be able to push them with `[quick ci]` in the knowledge that a full run will be needed before being able to actually merge (which can then be achieved by simply amending the last commit message).  Let's leave it in for now and see how it pans out.